wiki/Beatmap/Beatmap_description/th.md

```diff
@@ -0,0 +1,21 @@
+# คำอธิบายบีทแมป
+
+**คำอธิบายบีทแมป** เป็นพื้นที่เล็ก ๆ ใต้หน้าของ [บีทแมพ](/wiki/Beatmap) ที่ให้ข้อมูลเกี่ยวกับบีทแมปแก่ผู้เล่นโดยใช้ [BBCode](/wiki/BBCode) โดยปกติแล้วจะเน้นข้อเท็จจริงบางอย่างเกี่ยวกับบีทแมปผู้สร้างที่ถือว่ามีความสำคัญเพียงพอ หรือใช้เพื่อโฆษณาโปรเจคบีทแมพอื่น ๆ
+
+## การใช้และข้อมูล
+
+คำอธิบายยอดนิยมของบีทแมพที่ถูก[จัดอันดับ](/wiki/Beatmap/Category#ranked) มักมีดังต่อไปนี้:
+
+- รายการ[ระดับความยากของแขก](/wiki/Beatmap/Guest_difficulty)และผู้สร้างทั้งหมด
```

Comment:
Anything but ความยากของแขก, please.
For your context, its a difficulty created by other mappers that are submitted to the original mapper
For example, Reform's Expert difficulty in Sotarks map. This means Reform sends his difficulty to Sotarks and his difficulty is called a "guest difficulty". Correct me if I'm wrong
https://osu.ppy.sh/wiki/en/Beatmap/Guest_difficulty

Reply:
how bout this?